app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  before_save ->{email.downcase!}
+
+  Validates :name, presence: true,
+    length: {minimum: Settings.digits.length_10,
+             maximum: Settings.digits.length_30}
+  Validates :email, presence: true, uniqueness: true,
+    length: {minimum: Settings.digits.length_10,
+             maximum: Settings.digits.length_30},
+    format: {with: Settings.user.email.regex}
```

Comment:
Em đã test thử trường hợp nhập sai format của email chưa gái 

```suggestion
    format: {with: Regexp.new(Settings.user.email.regex)}
```